File: src/main/java/net/hectus/neobb/turn/card_game/CTChest.java
package net.hectus.neobb.turn.card_game;

import net.hectus.neobb.player.NeoPlayer;
import org.bukkit.block.Block;

public class CTChest extends CardTurn {
    public CTChest(NeoPlayer player) { super(player); }
    public CTChest(Block data, NeoPlayer player) { super(data, player); }

    @Override
    public double damage() { return 5; }
}

File: src/main/kotlin/net/hectus/neobb/turn/card_game/CTChest.kt
package net.hectus.neobb.turn.card_game

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.player.NeoPlayer
import org.bukkit.block.Block

class CTChest(data: Block?, cord: Cord?, player: NeoPlayer?) : CardTurn(data, cord, player) {
    override val damage: Double = 5.0
}